cmd/xmidt-agent/config.go

```diff
@@ -173,6 +174,10 @@ type Storage struct {
 	Durable   string
 }
 
+type MockTr181 struct {
+	FilePath string
```

Comment:
I think we should have an `Enabled` field to turn this functionality on or off explicitly.

Reply:
oooh, I thought there was going to be some config for each environment / scenario that specified which wrphandlers we wanted.  Similar to java spring (sorry if that is an insulting comparison)

Reply:
Basically yes, like Java spring.  An `Enabled` field will make it easy to configure the module on or off because the structure will have default values & be present in all configuration.